### PR TITLE
fix: adding env variable, command, correcting image url [#7]

### PR DESCRIPTION
**Pull request by nimblenitin**
adding env variable, command, correcting image url.

### CODE REVIEW

File: examples/single-subscription/variables.tf [hunk 1]
```diff
@@ -56,23 +56,23 @@ variable "deploy_scanning" {
 variable "mode" {
   type        = string
   description = "mode"
-  default     = ""
+  default     = "default_value"
```
Comment (ramanan-ravi) [inline examples/single-subscription/variables.tf:59]: Make it `service`

File: examples/single-subscription/variables.tf [hunk 2]
```diff
@@ -56,23 +56,23 @@ variable "deploy_scanning" {
...
 }
 
 variable "mgmt-console-url" {
   type        = string
   description = "mgmt-console-url"
-  default     = ""
+  default     = "default_value"
```
Comment (ramanan-ravi) [inline examples/single-subscription/variables.tf:65]: Is this change required?

File: examples/single-subscription/variables.tf [hunk 3]
```diff
@@ -56,23 +56,23 @@ variable "deploy_scanning" {
...
   description = "mgmt-console-port"
-  default     = ""
+  default     = "default_value"
 }
 
 variable "deepfence-key" {
   type        = string
   description = "deepfence-key"
-  default     = ""
+  default     = "default_value"
```
Comment (ramanan-ravi) [inline examples/single-subscription/variables.tf:77]: Is this change required?

File: examples/single-subscription/variables.tf (hunk 1 above)
Comment (nimblenitin) [reply examples/single-subscription/variables.tf]: have updated in new push.

File: examples/single-subscription/variables.tf [hunk 4]
```diff
@@ -56,23 +56,23 @@ variable "deploy_scanning" {
...
   description = "mgmt-console-url"
-  default     = ""
+  default     = "default_value"
 }
 
 variable "mgmt-console-port" {
   type        = string
   description = "mgmt-console-port"
-  default     = ""
+  default     = "default_value"
```
[the remark this answers is not in the document]
Comment (nimblenitin) [reply examples/single-subscription/variables.tf]: have updated in new push.

File: examples/single-subscription/variables.tf (hunk 2 above)
Comment (nimblenitin) [reply examples/single-subscription/variables.tf]: not required. dropped in new push.

File: examples/single-subscription/variables.tf (hunk 3 above)
Comment (nimblenitin) [reply examples/single-subscription/variables.tf]: not required. dropped in new push.

**Review by ramanan-ravi — APPROVED** [review]
